Source/Core/Common/GekkoDisassembler.cpp

```diff
@@ -545,12 +584,12 @@ void GekkoDisassembler::bli(u32 in)
   if (d & 0x02000000)
     d |= 0xfc000000;
 
-  m_opcode = StringFromFormat("b%s", b_ext[in & 3]);
+  m_opcode = fmt::format("b{}", b_ext[in & 3]);
 
   if (in & 2)  // AA ?
-    m_operands = StringFromFormat("->0x%.8X", d);
+    m_operands = fmt::format("->0x{:.8X}", d);
   else
-    m_operands = StringFromFormat("->0x%.8X", *m_iaddr + d);
+    m_operands = fmt::format("->0x{:.8X}", *m_iaddr + d);
```

Comment:
I get crashes from this when looking at a disassembly, with the message "precision not allowed for this argument type".  It seems that `.8X` isn't allowed; looking elsewhere, shouldn't this be `08X` instead?

This isn't the only place that has a `.` like this:

* `ldst_offs` lines 156 and 159
* `imm` line 388
* `bc` lines 572 and 574
* `bli` lines 590 and 592

Reply:
Indeed, it should. This is a divergence from printf that I overlooked. This should be fixed in #8187